### PR TITLE
Changed the hash tag

### PR DESCRIPTION
**Pull request by pavangudiwada**
changed #newfirefox to #NewFirefox and changed a few strings

### CODE REVIEW

File: locales/te-IN.ftl [hunk 1]
```diff
@@ -1,7 +1,7 @@
 title = ఫైర్ఫాక్స్  క్వాంటం విడుదల ప్రచారం
-project-name = ఫైర్ఫాక్స్ క్వాంటం విడుదల
+project-name = Firefox Quantum విడుదల
 main-title =  ఫైర్ఫాక్స్ తిరుగుబాటు!
-main-description = క్రొత్త  ఫైర్ఫాక్స్ గురించి పదం వ్యాప్తి చేయడానికి మాకు సహాయం చెయ్యండి! మంగళవారం 14, 14:00 UTC , మీకు కొత  ఫైర్ఫాక్స్లొ నచిన వాటి గురించి #newfirefox ఉపయొగించి ట్వెట్ చెయండి లెదా ఫైర్ఫాక్స్  ట్వీట్లను రీట్వీట్ చెయండి 
+main-description = క్రొత్త  ఫైర్ఫాక్స్ గురించి పద వ్యాప్తి చేయడానికి మాకు సహాయం చెయ్యండి! మంగళవారం 14, 14:00 UTC ,మీకు క్రొత్త  ఫైర్ఫాక్స్లొ నచిన వాటి గురించి #NewFirefox ఉపయొగించి ట్వెట్ చెయండి లెదా ఫైర్ఫాక్స్  ట్వీట్లను రీట్వీట్ చెయండి
```
Comment (MichaelKohler) [inline locales/te-IN.ftl:4]: is the removed space here on purpose?

**Review by MichaelKohler — COMMENTED** [review]
Thanks, please see my comment.

**Comment by MichaelKohler** [comment]
Please also note that the file has been updated in the meantime. Can you check the new one and rebase this Pull Request to the latest version if necessary?

**Comment by pavangudiwada** [comment]
Right now I cant.. :(

**Comment by pavangudiwada** [comment]
If there is anything to change tell me ill do it ...and the space there was necessary. It just makes it look good..

**Pull request closed without merging**